Tests/UIKitTests/UIViewControllerExtensionsTests.swift

```diff
@@ -120,5 +120,76 @@ final class UIViewControllerExtensionsTests: XCTestCase {
         XCTAssertNil(childViewController.parent)
     }
 
+    #if os(iOS)
+    func testPresentPopover() {
+        let popover = UIViewController()
+        let presentingViewController = UIViewController()
+
+        //Putting the view controller in a window and running a RunLoop seems to be the only way to make
```

Comment:
A small and finicky note, but please put a space after the `/`s (in all the other comments too).
```suggestion
        // Putting the view controller in a window and running a RunLoop seems to be the only way to make
```

Reply:
@marcocapano Please look at all your other lines of comments where you've done this as well.